iotdb/src/main/java/org/apache/iotdb/db/engine/filenode/FileNodeProcessor.java

```diff
@@ -430,7 +430,7 @@ public FileNodeProcessorStatus getFileNodeProcessorStatus() {
   }
 
   /**
-   * execute filenode recovery.
+   * execute filenode recovery. and finally release the write lock this processor.
```

Comment:
I did not see any available unlocks in this method. Maybe you mean that the caller should unlock this processor?

Reply:
my fault. the javadoc is incorrect.